controllers/reconcile_cli.go

```diff
@@ -95,29 +92,26 @@ func (r *RabbitmqClusterReconciler) runSetPluginsCommand(ctx context.Context, rm
 		cmd := fmt.Sprintf("rabbitmq-plugins set %s", plugins.AsString(" "))
 		stdout, stderr, err := r.exec(rmq.Namespace, podName, "rabbitmq", "sh", "-c", cmd)
 		if err != nil {
-			logger.Error(err, "failed to set plugins",
-				"pod", podName,
-				"command", cmd,
-				"stdout", stdout,
-				"stderr", stderr)
-			return err
+			msg := "failed to set plugins on pod"
+			logger.Error(err, msg, "pod", podName, "command", cmd, "stdout", stdout, "stderr", stderr)
+			r.Recorder.Event(rmq, corev1.EventTypeWarning, "FailedReconcile", fmt.Sprintf("%s %s", msg, podName))
+			return fmt.Errorf(fmt.Sprintf("%s %s: %v", msg, podName, err))
```

Comment:
Nitty picky: should we just do `fmt.Errorf("string %s %s %v", a, b, c)` ?
I think we can avoid the call with `fmt.Sprintf()`

Reply:
you are right! `fmt.Sprintf()` is totally redundant here. updated!